epochStart/bootstrap/interface.go

```diff
@@ -0,0 +1,36 @@
+package bootstrap
+
+import (
+	"github.com/ElrondNetwork/elrond-go/data/block"
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/ElrondNetwork/elrond-go/sharding"
+)
+
+// MetaBlockInterceptorHandler defines what a component which will handle receiving the meta blocks should do
+type MetaBlockInterceptorHandler interface {
+	process.Interceptor
+	GetMetaBlock(target int, epoch uint32) (*block.MetaBlock, error)
```

Comment:
you might have interceptors with whitelist, only requested elements to enter. The problem, whitelisting is in the hardfork branch.

Reply:
We can refactor when that branch will arrive here.